src/blocks/plugins/ai-content/index.tsx

```diff
@@ -245,59 +259,26 @@ const AIToolbar = ({
 				)
 			}
 			<MenuGroup>
-				<ExternalLink className='o-menu-item-alignment' href="https://docs.themeisle.com/collection/1563-otter---page-builder-blocks-extensions" target="_blank" rel="noopener noreferrer">
-					{
-						__( 'Edit Prompts', 'otter-blocks' )
-					}
-				</ExternalLink>
-			</MenuGroup>
-			<MenuGroup>
-				<span className="o-menu-item-header o-menu-item-alignment">{__( 'Writing', 'otter-blocks' )}</span>
-				<ActionMenuItem actionKey='otter_action_generate_title' callback={onClose}>
-					{ __( 'Generate a heading', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_continue_writing' callback={onClose}>
-					{ __( 'Continue writing', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_summarize' callback={onClose}>
-					{ __( 'Summarize it', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_make_shorter' callback={onClose}>
-					{ __( 'Make it shorter', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_make_longer' callback={onClose}>
-					{ __( 'Make it longer', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_make_descriptive' callback={onClose}>
-					{ __( 'Make it more descriptive', 'otter-blocks' ) }
-				</ActionMenuItem>
-			</MenuGroup>
-			<MenuGroup>
-				<span className="o-menu-item-header o-menu-item-alignment">{__( 'Tone', 'otter-blocks' )}</span>
-				<ActionMenuItem actionKey='otter_action_tone_professional' callback={onClose}>
-					{ __( 'Professional', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_tone_friendly' callback={onClose}>
-					{ __( 'Friendly', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_tone_humorous' callback={onClose}>
-					{ __( 'Humorous', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_tone_confident' callback={onClose}>
-					{ __( 'Confident', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_tone_persuasive' callback={onClose}>
-					{ __( 'Persuasive', 'otter-blocks' ) }
-				</ActionMenuItem>
-				<ActionMenuItem actionKey='otter_action_tone_casual' callback={onClose}>
-					{ __( 'Casual', 'otter-blocks' ) }
-				</ActionMenuItem>
+				{
+					customActions.map( ( action, index ) => (
+						<ActionMenuItem key={index} actionIndex={index} actionKey={'otter_action_prompt'} callback={onClose}>
+							{ action.title }
+						</ActionMenuItem>
+					) )
+				}
 			</MenuGroup>
 			<MenuGroup>
 				<ActionMenuItem actionKey='otter_action_prompt' callback={onClose}>
 					{ __( 'Use as prompt', 'otter-blocks' ) }
 				</ActionMenuItem>
 			</MenuGroup>
+			<MenuGroup>
+				<ExternalLink className='o-menu-item-alignment' href="/wp-admin/admin.php?page=otter#integrations" target="_blank" rel="noopener noreferrer">
```

Comment:
Just to avoid any issues, we should not use relative URL but use the `window.themeisleGutenberg.optionsPath` instead.